src/shared/ui/widget-title/widget-title.tsx

```diff
@@ -4,7 +4,8 @@ import classNames from 'classnames/bind';
 
 import styles from './widget-title.module.scss';
 
-type WidgetTitleProps = HTMLAttributes<HTMLHeadingElement> & VariantProps<typeof widgetTitleVariants>;
+type WidgetTitleProps = HTMLAttributes<HTMLHeadingElement> &
+  VariantProps<typeof widgetTitleVariants>;
```

Comment:
```suggestion
type WidgetTitleProps = HTMLAttributes<HTMLHeadingElement>
& VariantProps<typeof widgetTitleVariants>;
```

Reply:
If the linter doesn't fix it, it's redundant.